main.go

```diff
@@ -220,40 +220,31 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 		Scheme: scheme,
 	}
 
-	config := config.Configuration{}
+	cfg := config.Configuration{}
 
 	if configFile == "" {
-		scheme.Default(&config)
+		scheme.Default(&cfg)
+		options = ctrl.Options{
+			Scheme:                 scheme,
+			Port:                   config.DefaultWebhookPort,
+			HealthProbeBindAddress: config.DefaultHealthProbeBindAddress,
+			MetricsBindAddress:     config.DefaultMetricsBindAddress,
+			LeaderElection:         false,
+		}
 	} else {
-		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(&config))
+		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(&cfg))
 		if err != nil {
 			setupLog.Error(err, "unable to load the config file")
 			os.Exit(1)
 		}
 
-		cfgStr, err := encodeConfig(&config)
+		cfgStr, err := encodeConfig(&cfg)
 		if err != nil {
 			setupLog.Error(err, "unable to encode config file")
 			os.Exit(1)
 		}
 		setupLog.Info("Successfully loaded config file", "config", cfgStr)
```

Comment:
We should also log when the configuration is just the default. It's useful for debugging.

Reply:
Makes sense.

Reply:
Done.